projects/aera/index.js

```diff
@@ -428,6 +439,100 @@ async function processSymbioticTvl(symbioticVaults, api) {
   })
 }
 
+async function processMorphoBlueTvl(morphoBlueVaults, api, MORPHO_BLUE) {
+  if (morphoBlueVaults.length === 0) return;
+
+  // Get the number of markets for each oracle
+  const marketLengths = await api.multiCall({
+    abi: abi.getMarketsLength,
+    calls: morphoBlueVaults.map(x => ({target: x[1], params: []}))
+  });
+
+
+  // Build calls to get each market by index
+  const marketCalls = [];
+  morphoBlueVaults.forEach(([vault, oracle], oracleIndex) => {
+    const length = marketLengths[oracleIndex];
+    for (let i = 0; i < length; i++) {
+      marketCalls.push({
+        target: oracle,
+        params: [i],
+        vault: vault,
+        oracleIndex: oracleIndex
+      });
+    }
+  });
+
+  if (marketCalls.length === 0) return;
+
+  // Get all markets from the Morpho Blue position oracles
+  const markets = await api.multiCall({ abi: abi.markets, calls: marketCalls });
+
+  const [marketInfos, marketParams] = await Promise.all(
+    [
+      api.multiCall({
+        abi: abi.market,
+        calls: markets.map(market => ({target: MORPHO_BLUE, params: [market[0]]}))
+      }),
+      api.multiCall({
+        abi: abi.idToMarketParams,
+        calls: markets.map(market => ({ target: MORPHO_BLUE, params: [market[0]] }))
+      })
+    ]
+  );
+
+  // Get positions for each vault across all markets
+  const positionCalls = [];
+  marketCalls.forEach((call, index) => {
+    const marketId = markets[index][0]; // Extract the market ID (bytes32)
+    positionCalls.push({
+      target: MORPHO_BLUE,
+      params: [marketId, call.vault]
+    });
+  });
+
+  const positions = await api.multiCall({
+    abi: abi.position,
+    calls: positionCalls
+  });
+
+  // Process positions and add to TVL
+  await Promise.all(positionCalls.map(async (call, index) => {
+    const [_, vault] = call.params;
+    const position = positions[index];
+    const marketParam = marketParams[index];
+    const marketInfo = marketInfos[index];
+
+    if (position.collateral > 0) {
+      try {
```

Comment:
please remove any try/catch

Reply:
The issue is that in the case of RPC failure it is likely TVL will be lost and we wont have any way of monitoring. Is there another way to distinguish between erc4626 and other?